doc/api/buffer.md

```diff
@@ -163,23 +169,23 @@ The character encodings currently supported by Node.js include:
 
 * `'ucs2'` - Alias of `'utf16le'`.
 
-* `'base64'` - Base64 string encoding. When creating a buffer from a string,
+* `'base64'` - Base64 encoding. When creating a `Buffer` from a string,
   this encoding will also correctly accept "URL and Filename Safe Alphabet" as
   specified in [RFC4648, Section 5].
 
 * `'latin1'` - A way of encoding the `Buffer` into a one-byte encoded string
   (as defined by the IANA in [RFC1345],
   page 63, to be the Latin-1 supplement block and C0/C1 control codes).
 
-* `'binary'` - Alias for `latin1`.
+* `'binary'` - Alias for `'latin1'`.
 
 * `'hex'` - Encode each byte as two hexadecimal characters.
 
 _Note_: Today's browsers follow the [WHATWG spec] which aliases both 'latin1' and
 ISO-8859-1 to Windows-1252. This means that while doing something like `http.get()`,
 if the returned charset is one of those listed in the WHATWG spec it's possible
-that the server actually returned `win-1252` encoded data, and using `latin1`
-encoding may incorrectly decode the graphical characters.
+that the server actually returned Windows-1252 encoded data, and using `'latin1'`
```

Comment:
Hrmm... well the reason I changed this was more to make a visual difference since this is where we're discussing the valid character encodings (which are styled with backticks) supported by node. Would keeping 'win-1252', but unstyled be ok?